### ISSUE TITLE
Failing sending messages without mobile data

### ISSUE THREAD

**Issue by mauricioduarte01** [ubports/telegram-app#99]
Hello,

I am experiencing an issue (probably already known). when I want to send a message and suddenly I lost my 3G/4G data connection, the message just get stuck and is never sent, even after I get the data connection back. so I have to rewrite it again. 

Thanks.

Keep up the good work!

**Comment by Flohack74** [comment]
Yes this happens always also when transitioning from WiFi to network "in the wrong moment". Sometimes it works. I did not find a pattern so far. But clearly telegram needs a unsend messages queue for that issues. Also in this case it can completly loose sync with the server, then you dont get new messages also.

**Comment by mauricioduarte01** [comment]
Thanks a lot for your quick response Florian, I appreciate it a lot. 
We hope you guys can find any solution about this in the future. 
Everything else is working just great with the app.
If there is anything else I can do to help you, I am here.
